sunpy/map/tests/test_plotting.py

```diff
@@ -82,12 +82,12 @@ def test_peek_grid_aia171(aia171_test_map):
 
 @figure_test
 def test_peek_grid_spacing_aia171(aia171_test_map):
-    aia171_test_map.peek(draw_grid=(5, 5) * u.deg)
+    aia171_test_map.peek(draw_grid=(10, 5) * u.deg)
 
 
 @figure_test
 def test_peek_limb_aia171(aia171_test_map):
-    aia171_test_map.peek(draw_limb=True)
+    aia171_test_map.peek(draw_limb=False)
```

Comment:
Why these changes?

Reply:
Just to break the tests, so I could see if the HTML worked.

Reply:
👍 will leave this un-resolved so we don't forget to remove